bluesky/callbacks/best_effort.py

```diff
@@ -529,6 +529,56 @@ def stop(self, doc):
         self.check_visibility()
         super().stop(doc)
 
+class HeadingPrinter(Callback):
+    def __init__(self, start_doc):
+        self._start_doc = start_doc
+        # Print heading.
+        tt = datetime.fromtimestamp(self._start_doc['time']).utctimetuple()
+        print("Transient Scan ID: {0}     Time: {1}".format(
+            self._start_doc['scan_id'],
+            time.strftime("%Y/%m/%d %H:%M:%S", tt)))
+        print("Persistent Unique Scan ID: '{0}'".format(
+            self._start_doc['uid']))
+
+
+class BaselinePrinter(Callback):
+    def __init__(self, start_doc, file=sys.stdout):
+        # We accept a start_doc but discard it.
+        ...
+        self._descriptors = {}
+        self._baseline_toggle = True
+        self._file = file
+    def descriptor(self, doc):
+        # TODO Check the stream name of this descriptor using doc['name']
```

Comment:
Can you remove this comment and the `...`? The idea has been implemented, so it is no longer "TO DO", and the code is self-explanatory so a comment is not necessary.